packages/core/admin/ee/server/controllers/admin.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { env } = require('@strapi/utils');
+const { getService } = require('../../../server/utils');
+
+module.exports = {
+  async licenseLimitInformation() {
+    const permittedSeats = strapi.ee.licenseInfo.seats;
+    if (!permittedSeats) {
+      return;
+    }
+
+    let shouldNotify = false;
+    let licenseLimitStatus = null;
+    let enforcementUserCount;
+
+    const currentActiveUserCount = await getService('user').getCurrentActiveUserCount();
+
+    const data = await getService('user').getDisabledUserList();
+
+    if (data.value) {
+      const eeDisabledUsers = JSON.parse(data.value);
+      enforcementUserCount = currentActiveUserCount + eeDisabledUsers.length;
+    } else {
+      enforcementUserCount = currentActiveUserCount;
+    }
+
+    if (enforcementUserCount > permittedSeats) {
+      shouldNotify = true;
+      licenseLimitStatus = 'OVER_LIMIT';
+    }
+
+    if (enforcementUserCount === permittedSeats) {
+      shouldNotify = true;
+      licenseLimitStatus = 'AT_LIMIT';
+    }
+
+    return {
+      data: {
+        enforcementUserCount,
+        currentActiveUserCount,
+        permittedSeats,
+        shouldNotify,
+        shouldStopCreate: currentActiveUserCount >= permittedSeats,
```

Comment:
Minor thing, I hate to be the guy :P , but do you think this variable could be renamed to something that is more representative?
`maxActiveUserLimitReached` / `userLimitExceeded` / `reachedMaxUserCapacity`

wdyt?

Reply:
Also wondering, what do we do if the user count is actually greater than the permitted seats? Do we allow that to happen?

Reply:
Could do, don't really see any reasons why not